src/components/Form2DataSection.js

```diff
@@ -37,8 +37,8 @@ class Form2DataSection extends React.Component {
                         <div className="row proceeding-style">
                             <div className="col-lg-2 col-md-2 col-sm-6 col-xs-6 respondent-name-label">
                                 <div style={{whiteSpace: 'nowrap'}}>
-                                    Respondent name(s) &nbsp;
-                                    <i className="fa fa-question-circle" aria-hidden="true" data-tip="Select the names of all those responding to the appeal"></i>
+                                    Respondent's name &nbsp;
```

Comment:
You overwrote my change here.

Reply:
changing back to "Respondent name(s) &nbsp;"...

Reply:
new commit added